ui/apps/platform/src/types/roleResources.ts

```diff
@@ -45,4 +45,5 @@ export type ResourceName =
     | 'ScannerDefinitions'
     | 'SensorUpgradeConfig'
     | 'ServiceIdentity'
+    | 'ClusterCVE'
```

Comment:
Ivan, although not a blocker to merge, it sometimes reduces merge conflicts to add in alphabetical order instead of at the end of a list (in this case in the sub-list of resources under To-be-deprecated comment. On the other side of the coin, the goal is to add fewer resources in the future.

As a follow up, I will add a comment in this file similar to https://github.com/stackrox/stackrox/blob/master/central/role/resources/list.go#L26
```text
// KEEP THE FOLLOWING LIST SORTED IN LEXICOGRAPHIC ORDER (case-sensitive).
```